File: python/q10.py
#!/usr/bin/python3

def isPrime(n):
    for i in range(2,int(n**0.5)+1):
        if n%i==0:
            return False
    return True
total=0
for i in range(2,2000000):
    if isPrime(i):
        total += i
print(total)

File: go/q10.go
package main
import("fmt";"github.com/daniel-willett/common-go-methods")

func main(){
	var total int = 0
	for n:=2; n<2000000; n++{
		if common.IsPrime(n)==true{
			total+=n
		}
	}
	fmt.Println(total)
}
